test/libsolidity/semanticTests/array/copying/copy_function_internal_storage_array.sol

```diff
@@ -18,6 +18,6 @@ contract C {
 // compileViaYul: also
 // ----
 // test() -> 7
-// gas irOptimized: 126552
+// gas irOptimized: 124080
```

Comment:
@cameel I checked the assembly. The difference is that the old version copies element by element and the new version copies slot by slot, as expected.